Dockerfile

```diff
@@ -67,6 +67,11 @@ RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key
   apt-get -yq install google-chrome-stable && \
   rm -r /etc/apt/sources.list.d/google.list
 
+# Support Web UI LTS2019
+RUN apt-get install -yq  && \
```

Comment:
Aren't we missing a package name after -yq?

Reply:
🤦  of course we are, copy and paste at its best. Thanks

Reply:
Done 👍